### PR TITLE
Add JAG

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://gitlab.com/coringao/jag/-/issues/4
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://gitlab.com/coringao/jag/-/merge_requests/2

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build com.gitlab.coringao.JAG

**Comment by Mailaender** [comment]
bot, build com.gitlab.coringao.JAG

File: jag.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">jag.desktop</id>
```
Comment (hfiguiere) [inline jag.appdata.xml:3]: app-id doesn't match.

**Comment by Mailaender** [comment]
bot, build com.gitlab.coringao.JAG

**Comment by Mailaender** [comment]
bot, build com.gitlab.coringao.JAG

File: com.gitlab.coringao.JAG.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+app-id: com.gitlab.coringao.JAG
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: jag
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
```
Comment (barthalion) [inline com.gitlab.coringao.JAG.yml:9]:
```suggestion
  - --socket=x11
```

Doesn't launch on Wayland.

**Comment by Mailaender** [comment]
bot, build com.gitlab.coringao.JAG

**Pull request closed without merging**